ocw/lib/EC2.py

```diff
@@ -334,3 +337,48 @@ def cleanup_images(self, valid_period_days):
                     else:
                         self.log_dbg("Delete image '{}' (ami:{})".format(img['Name'], img['ImageId']))
                         self.ec2_client(region).deregister_image(ImageId=img['ImageId'], DryRun=False)
+
+    def cleanup_k8s_jobs(self):
+        self.create_credentials_file()
+        subprocess.call(["aws", "sts", "get-caller-identity"])
+
+        clusters = {}
+        for region in self.cluster_regions:
+            response = self.eks_client(region).list_clusters()
+            if 'clusters' in response and len(response['clusters']) > 0:
+                clusters[region] = []
+                for cluster_name in response['clusters']:
+                    subprocess.call(["rm", "-r", "~/.kube"])
+                    subprocess.call(["aws", "eks", "update-kubeconfig", "--region", region, "--name", cluster_name])
+
+                    config.load_kube_config()
+                    k8s_client = client.BatchV1Api()
```

Comment:
please check `eks_client` , `ec2_client` and `ec2_resource` - this should be done in the same way

Reply:
done